frontend/src/metabase/admin/tasks/containers/Help.jsx

```diff
@@ -127,6 +127,14 @@ export default class Help extends Component {
           <AdminHeader title={t`Diagnostic Info`} className="mb2" />
           <p>{t`Please include these details in support requests. Thank you!`}</p>
           <InfoBlock>{detailString}</InfoBlock>
+          <div className="text-medium text-bold text-uppercase">{t`Advanced Details (click to download)`}</div>
```

Comment:
```suggestion
          <div className="text-medium text-bold text-uppercase py2">{t`Advanced Details (click to download)`}</div>
```

Reply:
You just want some spacing around this label?

Reply:
Yes.  Thanks, that seemed to do it.

<img width="446" alt="Screen Shot 2021-05-03 at 9 43 22 AM" src="https://user-images.githubusercontent.com/3521562/116891193-09d11780-abf4-11eb-8fa0-a2e6cb23e57c.png">